docs/alerts/creating-experience-alerts.md

```diff
@@ -1,11 +1,39 @@
 title: Creating Experience Alerts
 description: Step-by-step alert creation instructions for Experience
 
-Experience App allows you to create alert rules based on [Apdex score](https://en.wikipedia.org/wiki/Apdex) of Page Loads, HTTP Requests and On-Page Transactions.
+Each Experience App metric that you can create alert rules on has a bell icon, when clicked, shows a dropdown menu listing the metrics that you can choose. 
 
-![image alt text](../images/alerts/image_22.png)
+![Bell Icon for Page Load](../images/alerts/experience_alert_bell.png)
 
-You can set condition based on standard Apdex user satisfaction measurement:
+The supported Experience metrics are:
+
+- Apdex score for Page load
+- Apdex score for HTTP requests  
+- Apdex score for on page transactions
+- First contentful paint time
+- First paint (FP) time
+- HTTP requests count
+- HTTP requests load time
+- Long tasks count
+- Long tasks duration
+- Page load backend load time
+- Page load frontend load time
+- Page load time
+- HTTP resource load count
+- HTTP resource load time
+- HTTP resource transfer size
+- User sessions count
+- Users count
+
+The above metrics can also be chosen from in the alert rule creation dialog:
+
+![Experience Alert Metric](../images/alerts/experience_alert_metrics.png)
+
+While creating an alert you'll get a chart preview of the alert threshold or anomaly. For example, to set up a new alert rule for that would be triggered for page load time higher than 4500 milliseconds on average this is how the preview screen would look like:
```

Comment:
For example, to set up a new alert rule for that would be triggered for page load time higher than 4500 milliseconds on average this is how the preview screen would look like:

Something's wrong with this sentence...

Reply:
Adjusted.